src/client/components/app/Navbar/index.tsx

```diff
@@ -109,9 +117,22 @@ export const Navbar = ({
     Icon: getNetworkIcon(network),
   }));
 
+  const secondTitleEnabled = !!subTitle?.length;
+
+  const vaultText = (
+    <>
+      &nbsp;/&nbsp;<StyledText>{subTitle}</StyledText>
+    </>
+  );
+
   return (
     <StyledNavbar className={className}>
-      {title && <StyledText>{title}</StyledText>}
+      {title && (
+        <StyledText toneDown={secondTitleEnabled}>
+          {title}
+          {secondTitleEnabled && vaultText}
```

Comment:
question: Will this just appear next to one another? Is that the intention?

Reply:
yes.
<img width="373" alt="image" src="https://user-images.githubusercontent.com/78362532/163129521-c832b881-15ea-4ce6-a0ce-bf3e6f4cba5b.png">